project2/README.md

```diff
@@ -53,14 +53,14 @@ pipenv run lint
 `py/main.py` takes an AllRecipes url from standard input and prints a transformed recipe to standard ouput. Eg:
 
 ```sh
-python py/main.py <transform_code> < address
+python py/main.py {transform_code} < address
```

Comment:
for clarity.

people were missing the `<` operator so hopefully this is more contrast